internal/driver/gomobile/clipboard_ios.go

```diff
@@ -0,0 +1,30 @@
+// +build ios
+
+package gomobile
+
+/*
+#cgo CFLAGS: -x objective-c
+#cgo LDFLAGS: -framework Foundation -framework UIKit -framework MobileCoreServices
+
+#include <stdlib.h>
+
+void setClipboardContent(char *content);
+char *getClipboardContent();
+*/
+import "C"
+import "unsafe"
+
+// Content returns the clipboard content for iOS
+func (c *mobileClipboard) Content() string {
+	content := C.getClipboardContent()
+
+	return C.GoString(content)
```

Comment:
does content need to be freed?

Reply:
Nah, it comes from https://developer.apple.com/documentation/foundation/nsstring/1411189-utf8string?language=objc - the content lifecycle is managed by the owning NSString.